### PR TITLE
Fix tickCount issue #175

### LINKED ISSUE #175: YAxis tickCount not working as expected
Hi, I found that when I change YAxis `tickCount` prop it works improperly. Here is a line chart with a brush. 

Here `tickCount` is not set for both chart and brush
image

Here `tickCount` is set to 5 for both charts
image

Here `tickCount` is set to 2 for both charts
image

As you can see it works weirdly for both charts. For the bottom one, it distributes ticks unevenly. I expected the only two ticks to be at the top and the bottom most parts of the scale. For the top one, it just doesn't make any sence.

I am using the latest version `0.12.8`
Here is the component's [source code](https://pastebin.com/Js5zhzVj)

### PR DESCRIPTION
**Pull request by sartaj10**
Fix #175

### CODE REVIEW

File: src/components/YAxis.js [hunk 1]
```diff
@@ -174,10 +175,21 @@ export default class YAxis extends React.Component {
         let axisGenerator;
         const axis = align === "left" ? axisLeft : axisRight;
         if (this.props.type === "linear" || this.props.type === "power") {
-            if (this.props.height <= 200) {
-                if (this.props.tickCount > 0) {
+            if (this.props.tickCount > 0) {
+                const stepSize = (this.props.max - this.props.min) / (this.props.tickCount - 1);
+                axisGenerator = axis(scale)
+                    .tickValues(range(this.props.min, this.props.max + 1, stepSize))
```
Comment (pjm17971) [inline src/components/YAxis.js]: Why `this.props.max + 1`? It seems like this could be very significant depending on what the overall scale is?

Comment (sartaj10) [inline src/components/YAxis.js]: Fixed.

**Comment by pjm17971** [comment]
This tickCount PR seems to have the alpha pond changes in it as well?

**Comment by dchhetri-ev** [comment]
Bumping this! Best.

**Pull request closed without merging**